<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingWindow</class>
 <widget class="QDialog" name="settingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="cookieLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>15</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cookieLineEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>56</y>
     <width>472</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="exceptOrderLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>180</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>355</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageInformationLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>312</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="fonts/resources.qrc">:/logo/information.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="informationTextLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="changeCookieLabel">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>335</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>217</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MyTextEdit" name="exceptOrdersTextEdit" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="phpSessidLabel">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>100</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phpSessLineEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>137</y>
     <width>472</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTextEdit</class>
   <extends>QWidget</extends>
   <header>mytextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fonts/resources.qrc"/>
 </resources>
 <connections/>
</ui>
